test/helpers/docker.go

```diff
@@ -32,9 +32,10 @@ func (s *SSHMeta) ContainerExec(name string, cmd string, optionalArgs ...string)
 
 // ContainerCreate is a wrapper for `docker run`. It runs an instance of the
 // specified Docker image with the provided network, name, and options.
```

Comment:
If you add a new argument to the function the comment should be updated.

Reply:
done